App/src/main/resources/com/gluonhq/chat/views/chat.css

```diff
@@ -174,14 +174,22 @@
     -fx-padding: 10;
 }
 
-.chat-message-text {
+.chat-message-sent {
     -fx-background-color: -chat-bubble-background;
-    -fx-padding: 10;
-    -fx-background-radius: 0 .7em .7em .7em;
+}
+
+.chat-message-received {
+    -fx-background-color: #e9e9e9;
```

Comment:
This new color should have a name, like `-chat-received-background` or similar, and be added to the root in `styles.css`, also it should work with both light and dark themes, so probably it needs the `-light` variant.

Reply:
Sure

Reply:
Will include this in an upcoming PR